application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -58,18 +59,17 @@ public void runRoutine(JDA jda) {
             .where(PENDING_REMINDERS.REMIND_AT.lessOrEqual(now))
             .stream()
             .forEach(pendingReminder -> {
-                sendReminder(jda, pendingReminder.getId(), pendingReminder.getChannelId(),
-                        pendingReminder.getAuthorId(), pendingReminder.getContent(),
-                        pendingReminder.getCreatedAt());
+                sendReminder(jda, Reminder.from(pendingReminder));
```

Comment:
pass the pendingReminder all along

Reply:
gonna double check if thats possible. the `pendingReminder` itself might still be tied to the DB and hold the resource open.

Reply:
if that's not possible i was thinking instead of using `from` method inside record i think it's better to just use `record` otherwise it ends up with really longer calls like `reminder.pendingReminder().getId()`, earlier it was something like `reminder.Id()` with just records.

Reply:
i can't see a reason y it is not possible, do u get an error or something? I believe i've that in this repo somewhere

Reply:
for the record, just passed `pendingReminder` and it works